### DISCUSSION TITLE
Wireless Android debugging?

### DISCUSSION THREAD

**Discussion by flintcreek** [beeware/briefcase#2008] [Q&A]
Is it possible to connect via Wi-Fi instead of USB to debug/run a Beeware app on an Android 13 device?  My android device has wi-fi debugging enabled, and it is paired with my Mac, but when I run "briefcase run android" it doesn't show up in the list of devices.  I'm thinking I have to connect to the ip:port first, but not sure how or if that works with briefcase.

**Comment by rmartin16** [comment]
I was able to debug wirelessly.

![image](https://github.com/user-attachments/assets/08e2e7d3-687f-4ff7-88c8-b61922ea83e6)

These [directions](https://www.androidpolice.com/use-wireless-adb-android-phone/) worked for me; once I paired my desktop with my Android device and connected, it showed up in the list.

**Reply by freakboy3742** [reply]
Mildly annoying that this isn't mentioned in the *Android developer guide*...

**Comment by flintcreek** [comment]
Thanks, rmartin16.  That worked.
